lib/twitter-ads/client.rb

```diff
@@ -3,7 +3,7 @@
 
 module TwitterAds
 
-  API_VERSION = '3'.freeze
+  API_VERSION = '4'.freeze
```

Comment:
Lets keep this `3` for now, once all the other changes have been merged in, we'll make a separate version bump update.